packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedback.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { ButtonIcon, ButtonPrimary, Flex, Link, Text } from 'design';
+import Validation from 'shared/components/Validation';
+import { Close } from 'design/Icon';
+import { ShareFeedbackForm } from './ShareFeedbackForm';
+import { ShareFeedbackFormValues } from './types';
+
+interface ShareFeedbackProps {
+  onClose(): void;
+}
+
+export function ShareFeedback(props: ShareFeedbackProps) {
+  const [formValues, setFormValues] = useState<ShareFeedbackFormValues>({
+    feedback: '',
+    email: '',
+    newsletterEnabled: false,
+    salesContactEnabled: false,
+  });
+
+  return (
+    <Flex bg="primary.main" p={3} borderRadius={3}>
+      <Validation>
+        {({ validator }) => (
+          <Flex
+            flexDirection="column"
+            as="form"
+            onSubmit={e => {
+              e.preventDefault();
+              if (validator.validate()) {
+                console.log('submit', formValues); //TODO (gzdunek): connect to a real service
```

Comment:
Any ideas on what it should say after submitting? Something like "Feedback submitted!" should be fine for now I guess?